README.md

```diff
@@ -199,6 +199,58 @@ startNewSession(options).then((session) => {
 });
 ```
 
+**Comm**
+
+```typescript
+import {
+  getKernelSpecs, startNewKernel, CommManager
+} from 'jupyter-js-services';
+
+var BASEURL = 'http://localhost:8888';
+var WSURL = 'ws://localhost:8888';
+
+// Create a comm from the server side.
+//
+// get info about the available kernels and connect to one
+getKernelSpecs(BASEURL).then((kernelSpecs) => {
+  var options = {
+    baseUrl: BASEURL,
+    wsUrl: WSURL,
+    name: kernelSpecs.default
+  }
+  startNewKernel(options).then((kernel) => {
+    var manager = new CommManager(kernel);
+      manager.connect('test').then((comm) => {
+        comm.open('initial state');
+        comm.send('test');
+        comm.close('bye');
+        done();
+      });
+    });
+  });
+});
```

Comment:
One can return Promises from a `then` handler - which reduces indentation and makes things easier to read:

```typescript
getKernelSpecs(BASEURL).then(kernelSpecs => {
  return startNewKernel({
    baseUrl: BASEURL,
    wsUrl: WSURL,
    name: kernelSpecs.default,
  });
}).then(kernel => {
  var manager = new CommManager(kernel);
  return manager.connect('test');
}).then(comm => {
  comm.open('initial state');
  comm.send('test');
  comm.close('bye');
});
```